app/command.go

```diff
@@ -39,8 +39,13 @@ func GetCommandProvider(name string) CommandProvider {
 	return nil
 }
 
-func (a *App) CreateCommandPost(post *model.Post, teamId string, response *model.CommandResponse) (*model.Post, *model.AppError) {
-	post.Message = model.ParseSlackLinksToMarkdown(response.Text)
+func (a *App) CreateCommandPost(post *model.Post, teamId string, response *model.CommandResponse, isCodeBlock bool) (*model.Post, *model.AppError) {
+	if isCodeBlock {
```

Comment:
I can't think of a better way to handle this special casing either. The one thing I would suggest is to name this parameter to be more generic, such as `skipSlackParsing` instead of `isCodeBlock`, since there could be other cases unrelated to the text being a code block where we might not want to do the Slack parsing

Reply:
Good idea, changed!